packages/react-storage/src/components/StorageBrowser/context/elements/definitions.ts

```diff
@@ -44,6 +44,38 @@ export type IconVariant =
 
 export type IconElementProps = React.ComponentProps<typeof IconElement>;
 
+export const DEFAULT_ICON_PATHS: Record<IconVariant, string> = {
```

Comment:
Let's move this in to a sibling _IconElement.tsx_ file where we can wire up a base `IconElement` component